database-population.js

```diff
@@ -0,0 +1,72 @@
+const faker = require('faker');
+const db = require('./database/reviews');
+
+const numUsers = 100;
+const numReviews = 500;
+
+const arrayOfAvatarURLs = ['https://s3-us-west-1.amazonaws.com/fec-reviews/1.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/2.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/3.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/4.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/5.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/6.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/7.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/8.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/9.jpg', 'https://s3-us-west-1.amazonaws.com/fec-reviews/10.jpg'];
+
+let arrayOfUsers = [];
+
+const generateRandomNumber = function (val) {
+    return Math.ceil(val * Math.random());
```

Comment:
Is this meant to include 0? (I think that makes it more useful for random indexing...or perhaps you could add an optional parameter to allow it to go either way, so it'd be function(upperLimit, lowerLimit = 0), and then you could return Math.floor(upperLimit * Math.random()) + lowerLimit)